src/Raven.Server/Web/Studio/Processors/AbstractBucketsHandlerProcessorForGetBucket.cs

```diff
@@ -77,12 +77,12 @@ public override void SetResponse(JsonOperationContext context, BlittableJsonRead
 
     public class BucketInfo : BucketStats
     {
-        public List<string> Documents;
+        public List<string> Items;
```

Comment:
@ml054 I've changed the name here to `Items` and it will list anything on this bucket